velox/functions/sparksql/Arithmetic.h

```diff
@@ -35,13 +35,23 @@ struct RemainderFunction {
     if (UNLIKELY(n == 0)) {
       return false;
     }
-    // std::numeric_limits<int64_t>::min() % -1 could crash the program since
-    // abs(std::numeric_limits<int64_t>::min()) can not be represented in
-    // int64_t.
-    if (UNLIKELY(n == 1 || n == -1)) {
-      result = 0;
+    if constexpr (std::is_floating_point_v<TInput>) {
```

Comment:
Could we extract them into another private function for floating types?

Reply:
Done~